tests/conditionalinfoflow.conf

```diff
@@ -0,0 +1,43 @@
+class infoflow
+
+sid kernel
+
+class infoflow
+{
+    hi_w
+    hi_r
+    super_r
+    super_w
+}
+
+type system;
+role system;
+role system types system;
+
+#################################################
+
+type src;
+type tgt;
+type flow_true;
+type flow_false;
+
+bool condition false;
+
+allow src self:infoflow super_w;
+
+if (condition) {
+    allow src flow_true:infoflow hi_w;
+    allow tgt flow_true:infoflow hi_r;
+}
+else {
+    allow src flow_false:infoflow hi_w;
+    allow tgt flow_false:infoflow hi_r;
+}
```

Comment:
It seems like this isn't testing the case where rules are deleted from an edge, but there are still remaining rules, so the Edge isn't deleted.